src/crypto/index.js

```diff
@@ -153,9 +157,10 @@ Crypto.prototype.init = async function() {
         };
 
         myDevices[this._deviceId] = deviceInfo;
-        this._sessionStore.storeEndToEndDevicesForUser(
+        this._deviceList.storeDevicesForUser(
             this._userId, myDevices,
```

Comment:
This & the one above get rid of the direct store access and go via the `DeviceList` class

edit: and also a couple below